test_distributions failures

currently getting these:

``` bash
$ nosetests
........................................................................................................................................................................................................................................FF...............F....................................................................................................................................................................................................
======================================================================
FAIL: grouped_distributions() should work even when there aren't
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mcdonadt/ResearchWork/software/scikit-bio/skbio/draw/tests/test_distributions.py", line 386, in test_grouped_distributions_insufficient_colors
    "There are not enough markers to uniquely "
AssertionError: 'There are not enough markers to uniquely represent each distribution in your dataset. You may want to provide a list of markers that is at least as large as the number of distributions in your dataset.\ntight_layout : falling back to Agg renderer' != 'There are not enough markers to uniquely represent each distribution in your dataset. You may want to provide a list of markers that is at least as large as the number of distributions in your dataset.'

======================================================================
FAIL: grouped_distributions() should work even when there aren't
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mcdonadt/ResearchWork/software/scikit-bio/skbio/draw/tests/test_distributions.py", line 423, in test_grouped_distributions_insufficient_symbols
    "There are not enough markers to uniquely "
AssertionError: 'There are not enough markers to uniquely represent each distribution in your dataset. You may want to provide a list of markers that is at least as large as the number of distributions in your dataset.\ntight_layout : falling back to Agg renderer' != 'There are not enough markers to uniquely represent each distribution in your dataset. You may want to provide a list of markers that is at least as large as the number of distributions in your dataset.'

======================================================================
FAIL: Test setting a figure size that has really long labels.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mcdonadt/ResearchWork/software/scikit-bio/skbio/draw/tests/test_distributions.py", line 626, in test_set_figure_size_long_labels
    "Warning: could not automatically resize plot to make room "
AssertionError: "tight_layout : falling back to Agg renderer\nWarning: could not automatically resize plot to make room for axes labels and plot title. This can happen if the labels or title are extremely long and the plot size is too small. Your plot may have its labels and/or title cut-off. To fix this, try increasing the plot's size (in inches) and try again." != "Warning: could not automatically resize plot to make room for axes labels and plot title. This can happen if the labels or title are extremely long and the plot size is too small. Your plot may have its labels and/or title cut-off. To fix this, try increasing the plot's size (in inches) and try again."

----------------------------------------------------------------------
Ran 446 tests in 22.552s

FAILED (failures=3)
```

--- a/skbio/draw/tests/test_distributions.py
+++ b/skbio/draw/tests/test_distributions.py
@@ -14,6 +14,8 @@ from StringIO import StringIO
 from unittest import TestCase, main
 
 import numpy as np
+from matplotlib import use
+use('Agg', warn=False)
 import matplotlib.pyplot as plt
 
 from skbio.draw.distributions import (boxplots, grouped_distributions,

